### PR TITLE
Propagate generics to functions, don't serialize overriden functions

### PR DESCRIPTION
**Pull request by BarkingBad**

### CODE REVIEW

File: dokka-common/db-generator/src/main/kotlin/org/virtuslab/inkuire/plugin/dbgenerator/InkuireDbGeneratorDokkaPlugin.kt [hunk 1]
```diff
@@ -15,4 +15,6 @@ class InkuireDbGeneratorDokkaPlugin : DokkaPlugin() {
     val renderer by extending {
         CoreExtensions.renderer providing ::InkuireJustDbRenderer override dokkaBase.htmlRenderer
     }
+
+    val x = listOf(1, 2, 3).get(1)
```
Comment (KacperFKorban) [inline dokka-common/db-generator/src/main/kotlin/org/virtuslab/inkuire/plugin/dbgenerator/InkuireDbGeneratorDokkaPlugin.kt:19]: rm?

File: engine/src/main/scala/org/virtuslab/inkuire/engine/service/DefaultDokkaModelTranslationService.scala [hunk 2]
```diff
@@ -48,26 +48,19 @@ object DefaultDokkaModelTranslationService extends DokkaModelTranslationService
...
-                callableName = None,
-                original = {
-                  val array = f.getDri.getOriginal.split("/")
-                  array(2) = ""
-                  array.mkString(sep = "/") + "/"
-                }
-              )
-              .some
+      .orElse {
+//        if(f.getDri.getClassName == "List") println(f.getDri, translateDRI(f.getDri))
```
Comment (KacperFKorban) [inline engine/src/main/scala/org/virtuslab/inkuire/engine/service/DefaultDokkaModelTranslationService.scala:55]: rm

File: dokka-common/build.gradle.kts [hunk 3]
```diff
@@ -29,8 +29,8 @@ dependencies {
 
 publishing {
     publications {
-        register<MavenPublication>("inkuirePlugin") {
-            artifactId = "inkuire-plugin"
+        register<MavenPublication>("dokkaCommon") {
+            artifactId = "dokka-common"
```
Comment (pikinier20) [inline dokka-common/build.gradle.kts:33]: I think we shouldn't make artifactId with this name. It indicates that it's part of dokka, not Inkuire and may be misleading.

File: dokka-common/db-generator/build.gradle.kts [hunk 4]
```diff
@@ -17,6 +21,15 @@ dependencies {
     implementation(project(":dokka-common"))
 }
 
+publishing {
+    publications {
+        register<MavenPublication>("dbGenerator") {
+            artifactId = "db-generator"
+            from(components["java"])
```
Comment (pikinier20) [inline dokka-common/db-generator/build.gradle.kts:28]: same

File: common/build.gradle.kts [hunk 5]
```diff
@@ -25,7 +25,7 @@ tasks {
 
 publishing {
     publications {
-        register<MavenPublication>("inkuireCommon") {
+        register<MavenPublication>("common") {
```
Comment (pikinier20) [inline common/build.gradle.kts:28]: same

**Pull request merged**